projects/Mallard/src/components/front/layouts.ts

```diff
@@ -38,4 +60,38 @@ const fiveStoryPage: PageLayout = withSlots([
     },
 ])
 
-export { superHeroPage, threeStoryPage, fiveStoryPage }
+const sixStoryPage: PageLayout = withSlots([
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+    {
+        size: RowSize.row,
+        columns: [SmallItem],
+    },
+])
+
+export {
```

Comment:
why not `export const` each time instead?

Reply:
Just wading in here, I find `exports` at the bottom is a slightly easier pattern. It means you can scroll to the bottom of the file and determine the exports immediately. Just my tuppence!

Reply:
mere choice tbh i like them on the bottom better

Reply:
😳
not sure I find this easier personally, if you want to lint this I guess I'd be happy to go along with it.